<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfilingWidget</class>
 <widget class="QWidget" name="ProfilingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1401</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {

background-color: #E1F8E8;

}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Manage Residents</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">QFrame {

background-color: white;

}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Search:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="line_search">
              <property name="maximumSize">
               <size>
                <width>500</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>-1</pointsize>
                <underline>false</underline>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">/* Base QLineEdit styling */
QLineEdit {
    background-color: #fff;           /* white background */
    border: 1px solid #ccc;           /* light gray border */
    border-radius: 6px;               /* rounded corners */
    padding: 6px 8px;                 /* vertical/horizontal padding */
    font-size: 14px;                  /* sensible text size */
    color: #333;                      /* dark text */
}

/* Hover state: slightly darker border */
QLineEdit:hover {
    border: 1px solid #999;
}

/* Focus state: blue glow and stronger border */
QLineEdit:focus {
    border: 1px solid #409EFF;        /* blue accent */
    outline: none;                    /* remove default focus ring */
    box-shadow: 0 0 3px rgba(64,158,255,0.5);
}

/* Disabled state: muted look */
QLineEdit:disabled {
    background-color: #f5f5f5;
    border: 1px solid #e0e0e0;
    color: #999;
}

/* Placeholder text styling */
QLineEdit[echoMode=&quot;0&quot;] {            /* only applies when not password mode */
    qproperty-placeholderText: &quot;Enter textâ€¦&quot;;
}
QLineEdit::placeholder {
    color: #999;
    font-style: italic;
}
</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_5">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="maximumSize">
                  <size>
                   <width>30</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Age:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="combo_age"/>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="maximumSize">
                  <size>
                   <width>37</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Purok:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="combo_purok">
                 <item>
                  <property name="text">
                   <string>Purok Nangka</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Purok Lubi</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="maximumSize">
                  <size>
                   <width>37</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Voter:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="combo_registered">
                 <item>
                  <property name="text">
                   <string>Yes</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>No</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="addbutton">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>500</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Add New Resident</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableresidents">
           <property name="styleSheet">
            <string notr="true">QTableWidget {
    border: 1px solid #ccc;
    gridline-color: #eee;
    selection-background-color: #87CEFA;
    font-size: 14px;
}

QHeaderView::section {
    background-color: #f0f0f0;
    padding: 4px;
    font-weight: bold;
    border: 1px solid #dcdcdc;
}

 QTableView::item {
        /* only draw right &amp; bottom borders */
         border: 0.2px solid #333;
        
    }
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>1</number>
           </property>
           <property name="gridStyle">
            <enum>Qt::SolidLine</enum>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <row>
            <property name="text">
             <string>Trishtian</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>asd</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>ID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>First Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Middle Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Last Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Purok</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Cabs.qrc"/>
 </resources>
 <connections/>
</ui>
